Sources/Helpers.swift

```diff
@@ -1,5 +1,25 @@
 // MARK: - Functions
 
-func log(_ text: String) {
-  print("🔮 AFTERMATH: \(text)")
+enum LogType {
+  case normal
+  case warning
+  case error
+  case unknown
+}
+
+func log(_ text: String, type: LogType = .normal) {
+  var emoticon: String = ""
+
+  switch type {
+  case .warning:
+    emoticon = "⚠️ "
```

Comment:
I think the usage of dictionary here could save us some lines from this `switch case`. And an emoticon shou;d not contain space `"⚠️"`, we can add the space in the `print` statement